pkg/config/config.go

```diff
@@ -22,6 +22,8 @@ const (
 	REGISTRY_TLS_VERIFY = "registry_tls_verify"
 	KUBECONFIG          = "kubeconfig"
 	DEBUG               = "kfn_debug"
+	CONFIG				= "config"
+	NAMESPACE			= "namespace"
```

Comment:
Very minor, but the formatting here is weird.

Reply:
Solved in next ones :)